src/Makefile.am

```diff
@@ -2,8 +2,10 @@
 EXTRA_DIST = \
 	.indent.pro
 
-ubindir = ${prefix}/bin
-usbindir = ${prefix}/sbin
+prefix = /usr/local
+exec_prefix = ${prefix}
```

Comment:
Shouldn't this be `/usr`?

Reply:
Can you somehow indicate in the commit message that you have changed the default from `/usr` to `/usr/local`? This should warn developers and maintainers that the binary might not be installed where it was previously.